samples/cpp/benchmark_app/main.cpp

```diff
@@ -378,11 +378,19 @@ int main(int argc, char* argv[]) {
         for (auto& device : devices) {
             auto& device_config = config[device];
 
+            ov::hint::PerformanceMode ov_perf_hint = ov::hint::PerformanceMode::THROUGHPUT;
+
             // high-level performance modes
             if (!device_config.count(ov::hint::performance_mode.name())) {
-                device_config.emplace(ov::hint::performance_mode(get_performance_hint(device, core)));
+                auto ov_perf_hint = get_performance_hint(device, core);
+                if ((ov_perf_hint == ov::hint::PerformanceMode::LATENCY) ||
```

Comment:
Python should be updated to keep it aligned with C++

Reply:
updated

Reply:
I can't see any new commit

Reply:
@Wovchena Pushed commit to Github. Please review again.